api/base/pagination.py

```diff
@@ -168,6 +172,86 @@ def paginate_queryset(self, queryset, request, view=None):
             return super(JSONAPIPagination, self).paginate_queryset(queryset, request, view=None)
 
 
+class CursorPagination(pagination.CursorPagination):
+    page_size_query_param = 'page[size]'
+    max_page_size = MAX_PAGE_SIZE
+    total = 0
+
+    def get_response_dict(self, data, url):
```

Comment:
Is the `url` parameter being used by the function in a non-obvious way? It doesn't seem like it's being used to me.

Reply:
No, looks like copypasta

Reply:
I did see that the `reversed_url` for the embed view is being passed here (https://github.com/CenterForOpenScience/osf.io/pull/9134/files#diff-db89abfd7e02cdd0e3cabc7ec259acf1R212). Was the intention that the url be included in the response_dict or that the url is evaluated and the data returned is included in the `response_dict`?

Reply:
This is basically a copy of the JSONAPIPagination class but modified for cursor things. The response object needed URLs in the original in order to build the links to other parts of the pagination, but that's no longer necessary with cursor pagination, because you can't get to those pages any more. I just neglected to pull out the url parameter because the python linter isn't as strict as the javascript linter for this sort of thing, and the grey color difference in used and unused parameter was very subtle. :)

Reply:
And you're completely right about this, btdubs. I'm cleaning the url things out of the params now.